src/client/actions/PresenceUpdate.js

```diff
@@ -26,15 +26,15 @@ class PresenceUpdateAction extends Action {
       });
       this.client.emit(Events.GUILD_MEMBER_AVAILABLE, member);
     }
-    guild.presences._add(Object.assign(data, { guild }));
-    if (this.client.listenerCount(Events.PRESENCE_UPDATE) && member && !member.presence.equals(oldPresence)) {
+    const newPresence = guild.presences._add(Object.assign(data, { guild }));
+    if (this.client.listenerCount(Events.PRESENCE_UPDATE) && newPresence.equals(oldPresence)) {
```

Comment:
```suggestion
    if (this.client.listenerCount(Events.PRESENCE_UPDATE) && !newPresence.equals(oldPresence)) {
```
Probably got removed on accident